<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelDBViewer_widget</class>
 <widget class="QWidget" name="ModelDBViewer_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModelDBViewer_widget</string>
  </property>
  <widget class="QListWidget" name="catSelectListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>221</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="modelIdListWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>256</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="catNumLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Catgory Num</string>
   </property>
  </widget>
  <widget class="QLabel" name="modelNumLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Model Num</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
